src/main/webapp/WEB-INF/views/login.jsp

```diff
@@ -0,0 +1,36 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: adam
+  Date: 08.04.2021
+  Time: 16:31
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Strona logowania</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+<%@include file="header.jsp"%>
```

Comment:
Polecam https://www.baeldung.com/thymeleaf-in-spring-mvc aby nie wrzucać w każdy widok nagłówków